src/controllers/middlewares/verify-token.middleware.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response, NextFunction } from 'express';
+import jwt from 'jsonwebtoken';
+import { UnauthorizedError } from '../../errors/unauthorized.error';
+import { ForbiddenError } from '../../errors/forbidden.error';
+import config from '../../config';
+
+export const verifyTokenMiddleware = (req: Request, res: Response, next: NextFunction): void => {
+  if (req.url === '/userbase/login' && req.method === 'GET') {
```

Comment:
I don't think that hardcoding route path at the middleware is a good practice. It would work in this case but what if you have 10 routes that shouldn't be covered by `verifyTokenMiddleware`?